src/components/dropdownList/dropdownList.tsx

```diff
@@ -0,0 +1,28 @@
+import { NavLink } from "react-router-dom";
+import { CategoryInfo, categoryInfos } from "@/categoriesInfo";
+import styles from "./dropdownList.module.scss";
+
+interface ListHeader {
+  name: string;
+  className: string;
+  to: string;
+}
+
+function mapSublinks() {
+  return categoryInfos.map((link: CategoryInfo) => (
+    <li key={link.key} className={styles.listItem}>
+      <NavLink to={link.url}>{link.name}</NavLink>
+    </li>
+  ));
+}
```

Comment:
no need to create function for this operation

Reply:
mapping is inserted into return block